### PR TITLE
Proof read TAGS_FOR_RELATIVE_SETLOC

### LINKED ISSUE #263: Proof TAGS_FOR_RELATIVE_SETLOC
Proof read transcribed code of [TAGS_FOR_RELATIVE_SETLOC](//github.com/chrislgarry/Apollo-11/blob/master/Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc) against scans

Lines: 437
Page: 0027—0036

Reduced quality scans can be found [here][1]

[1]://ibiblio.org/apollo/ScansForConversion/Comanche055

### PR DESCRIPTION
**Pull request by Feliix42**
There was not much to fix in this part of the code except for a few comment fixes. But I am unsure as to whether the [subroutine calls on page 36](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0036.jpg) were left out intentionally or are just missing. Should they be added?

EDIT: Also, this closes #263

### CODE REVIEW

**Comment by wopian** [comment]
The subroutine lines on page 36 were generated by the assembler, so the page contains no source code to transcribe. 👍

File: Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc [hunk 1]
```diff
@@ -399,11 +399,11 @@ THISAXIS	=	UNITX
...
 DELAYNUM	EQUALS	THREE
 
-#********************************************************************************************************
+#***************************************************************************************************************
 
-# THE FOLLOWING ECADRS ARE DEFINED TO FACILITATE EBANK SWITCHING.  THEY ALSO MAKE IT EASIER FOR
+#   THE FOLLOWING ECADRS ARE DEFINED TO FACILITATE EBANK SWITCHING.  THEY ALSO MAKE IT EASIER FOR
 # ERASABLE CONTROL TO REARRANGE ERASABLE MEMORY WITHOUT DISRUPTING THE PROGRAMS WHICH SET EBANKS.
-# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).  EBANKX OF COURSE
+# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4.5.6.7).EBANKX OF COURSE
```
Comment (wopian) [inline Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc]: I am fairly certain these are commas that have been badly printed (quite common). They have a slight bump in the bottom left, compared to the periods that are symmetrical:



Better example at the top of the page:

Comment (Feliix42) [reply Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc]: I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Comment (Feliix42) [reply Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc]: I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Comment (Feliix42) [reply Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc]: I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Comment (Feliix42) [reply Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc]: I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Comment (Feliix42) [reply Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc]: I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Comment (Feliix42) [reply Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc]: I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Also sorry for my comment turning up several times, seems [Github is not feeling so good right now](https://status.github.com).

Comment (wopian) [reply Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc]: No problem, I can't apply the changes until they've resolved the issue either 😢

**Comment by wopian** [comment]
Thank you :tada:

**Pull request merged**